tutorials/vscCodeNavigation.md

```diff
@@ -0,0 +1,99 @@
+{% set title="VS Code - Code Navigation" %}
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
```

Comment:
Add here the alert about this being a community resource (see other similar tutorials for guidance)